PWGCF/TwoParticleCorrelations/Tasks/CMakeLists.txt

```diff
@@ -58,6 +58,11 @@ o2physics_add_dpl_workflow(neutron-proton-corr-zdc
                            PUBLIC_LINK_LIBRARIES O2Physics::AnalysisCore
                            COMPONENT_NAME Analysis)
 
+o2physics_add_dpl_workflow(ion-dihardon-cor
+                           SOURCES ionDihardonCor.cxx
```

Comment:
Please, don't use `ion` in the task name (nor in the source file), it is misleading
The same task can be used for pp collisions
`Dihardon` -> `DiHadron`